js/src/ux-capture.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable no-undef */
+/* eslint-env browser, node, amd */
```

Comment:
this comment is meant to disable the eslint rule `no-undef`. it's technically an eslint binding not a comment. let's add the original disable rule back and then add more comments if you like

Reply:
eslint-env is also an ESLint binding - it looks like it's a more specific way of disabling 'undefined var' errors for globals that are expected to be available in the specified environments. https://eslint.org/docs/user-guide/configuring#specifying-environments

Reply:
That was my thinking exactly, it keeps ESLint happy allowing globals specific to those environments.